package.json

```diff
@@ -63,6 +63,7 @@
     "prop-types": "15.8.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-helmet": "^6.1.0",
```

Comment:
[question] Could you please check your package-lock.json as we shouldn't have such big changes due to installing one package.